src/EnqueueIt.Dashboard/EnqueueIt.Dashboard.csproj

```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <Id>EnqueueIt.Dashboard</Id>
     <Title>EnqueueIt.Dashboard</Title>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
we try to support as old versions as possible

Reply:
Yep I understand, you can keep it in core 3.1, its just that the new System.Text.Json I added in the main EnqueueIt proj is not entirely tested or the support is unknown for netcore 3.1.
But I had to update Json because of vulnerabilities